src/UI/DappModal/hooks/useDappModal.ts

```diff
@@ -1,71 +1,29 @@
-import { useCallback, useEffect } from 'react';
-import { useIsDappModalVisible } from 'hooks';
-import { useDispatch, useSelector } from 'reduxStore/DappProviderContext';
-import { dappModalConfigSelector } from 'reduxStore/selectors/dappModalsSelectors';
-import {
-  setDappModalConfig,
-  setDappModalVisibility
-} from 'reduxStore/slices/dappModalsSlice';
-import {
-  DappModalConfig,
-  DappModalInteractionConfig
-} from '../dappModal.types';
+import { useEffect, useState } from 'react';
 
-export interface UseDappModalProps {
-  config?: DappModalInteractionConfig;
-  modalConfig?: DappModalConfig;
-}
+import { DappModalInteractionConfig } from '../dappModal.types';
 
-export const useDappModal = (props?: UseDappModalProps) => {
-  const dispatch = useDispatch();
-  const visible = useIsDappModalVisible();
```

Comment:
I think this was persisting the visibility in some cases

Reply:
It is not necessary to have one centralised modal managed with redux

Reply:
I think the idea was to have this type of modal visible once at a time

Reply:
This implies the component is also rendered once, like `<SignTransactionsModals />`, not multiple times in the app